standard/standard/clause_7_normative_text.adoc

```diff
@@ -94,1003 +133,1167 @@ The Basic and Advanced Standardization Targets differ in the level of options an
 ... Open-ended sequence of time-stamped GeoPoses: *Stream* Target
 
 [NOTE]
-The defintion of a reference frame by an external standard in *not* specified. GeoPose does use a three-part designation of an external frame specification using the three fields _authority_, _ID_, and _parameters_. The interpretation of the contents of these fields is outside the scope of GeoPose.
-
+The defintion of a reference frame by an external standard in *not* specified.
+GeoPose does use a three-part designation of an external frame specification using
+the three fields _authority_, _ID_, and _parameters_. The interpretation of the
+contents of these fields is outside the scope of GeoPose.
 
-===  Global Requirements (Normative)
+=== Global Requirements (Normative)
```

Comment:
Remove the word `(Normative)`. All content not marked informative is normative.

Reply:
Removed.

Reply:
Removed all `(Normative)` indicators.